Assets/Scripts/Mechanics/Health.cs

```diff
@@ -5,11 +5,15 @@
 
 namespace Platformer.Mechanics
 {
+    
     /// <summary>
     /// Represebts the current vital statistics of some game entity.
     /// </summary>
     public class Health : MonoBehaviour
     {
+
+        public PlayerController player;
```

Comment:
i don't think this circular reference is needed - player can already obtain a ref through using ```GetComponent```

Reply:
ok rectified